.gitignore

```diff
@@ -1,20 +1,27 @@
-/CMake*
-!/CMakeLists.txt
-/*.cmake
-/Makefile
-/gmock/
-/*.cpp
-/include/
-/ts/ts.qrc.depends
-/chewing-editor.desktop
-/chewing-editor
-/chewing-editor.1
-/chewing-editor.1.gz
-/chewing.json
-/*.a
-/run-test*
-/ts/*.qm
-/ui_*.h
-/Testing/
-/coverage*
-/chewing-editor.dSYM/
+CMake*
```

Comment:
And also break here....
On Mac OS X, the file system is case-insensitive, so the `cmake/` directory will be all ignored.